app/helpers/time_options_helper.rb

```diff
@@ -17,7 +17,7 @@ def format_time_12_hour_clock(datetime)
   end
 
   def format_full_date_and_time(datetime)
-    datetime.strftime('%A, %d %B %Y at %-l:%M %P')
+    datetime.strftime('%-l:%M%P on %-d %B %Y')
```

Comment:
Changing this format means that the scheduled date reads strangely now on the publish and delete page:

`Scheduled to be published on 11:26am on 20 April 2020`

Reply:
Good spot. Updated now to be `Scheduled to be published at 11:26am on 20 April 2020`